packages/react-pose-core/src/components/Transition/index.tsx

```diff
@@ -23,12 +23,14 @@ class Transition extends React.PureComponent<Props> {
 
   removeChild(key: string) {
     const { children, leaving } = this.state;
-    const { enterAfterExit } = this.props;
+    const { enterAfterExit, onRest } = this.props;
     if (!leaving.hasOwnProperty(key)) return;
```

Comment:
is hasOwnProperty check needed here? like - wouldnt a "simpler" property lookup be enough?

Reply:
The reason I renamed this `finishedLeaving` in `react-pose` (and will do the same here at some point) is values in `finishedLeaving` can either be `true` or `false`.

`false` indicates it's leaving but hasn't finished, `true` it's finished.

So the alternative is `leaving[key] === undefined` which would be fine too, but also bikeshedding at this point